src/django_upgrade/main.py

```diff
@@ -22,6 +23,11 @@ def main(argv: Sequence[str] | None = None) -> int:
     parser = argparse.ArgumentParser()
     parser.add_argument("filenames", nargs="*")
     parser.add_argument("--exit-zero-even-if-changed", action="store_true")
+    parser.add_argument(
+        "--version",
+        action="version",
+        version=metadata.version("django-upgrade"),
```

Comment:
I guess it's best to use `%(prog)s` in the version string, as the argparse docs demonstrate for `action='version'`.